### ISSUE TITLE
Corruption of Localizable.strings

### ISSUE THREAD

**Issue by sschizas** [ginowu7/CleanLocalizableExample#4]
Hello @ginowu7, 

I ran you script and the Localizable.strings (English-development language) is erased and becomes like this:

ˇ˛// Alerts related to messaging

Any thoughts what may be the problem?

**Comment by ginowu7** [comment]
Interesting, is there any comments in the file? The script does not take ignore comments as of yet.

**Comment by ginowu7** [comment]
One more thing to check is make sure the format of the localizable file is one entry per line, eg:
```
"MoreTableViewController:Apple" = "Apple";
"MoreTableViewController:Banana" = "Banana";
```

**Comment by sschizas** [comment]
@ginowu7 yes my .strings file has a lot of comments as it Is generated via genstrings util. This is the root of the problem?? Do you have any plans for improving your script for ignoring comments?
